### DISCUSSION TITLE
(Question) is it possible to set background color to hex string

### DISCUSSION THREAD

**Discussion by oong819** [Sketchware-Pro/Sketchware-Pro#646] [Q&A] [answered]
is it possible to set background color to hex string? Skerchware Pro currently only support set color using a "spinner"

**Comment by j-px** [comment]
That's what you use Custom Colors for.  
![Screenshot_20221118_080809_Sketchware Pro](https://user-images.githubusercontent.com/40118124/202642669-c35c38a8-10fb-4c4c-b7b0-fa51a9fc8972.png)

**Comment by oong819** [comment]
> That's what you use Custom Colors for. ![Screenshot_20221118_080809_Sketchware Pro](https://user-images.githubusercontent.com/40118124/202642669-c35c38a8-10fb-4c4c-b7b0-fa51a9fc8972.png)

i mean set color to a string variable

im trying to make a color picker app

**Comment by subhash16** [comment, marked as answer]
view_id.setBackgroundColor(Color.parseColor("Color Hex code"));

Replace view_id with your linear or anything
Replace colour Hex code with your code

**Comment by oong819** [comment]
> view_id.setBackgroundColor(Color.parseColor("Color Hex code"));
> 
> Replace view_id with your linear or anything Replace colour Hex code with your code

thanks!

**Comment by subhash16** [comment]
You can get in touch with me on WhatsApp

**Reply by DevVigilante** [reply]
I need a help please answer it😥😥.
https://github.com/Sketchware-Pro/Sketchware-Pro/discussions/665#discussion-4618562

**Reply by subhash16** [reply]
What type of help

**Reply by DevVigilante** [reply]
Click on https://github.com/Sketchware-Pro/Sketchware-Pro/discussions/665#discussion-4618562
